joblib/test/test_parallel.py

```diff
@@ -10,6 +10,10 @@
 import sys
 import io
 import os
+import subprocess
+from threading import Timer
+import re
```

Comment:
I would switch the 2 lines above. `import re` first then `from threading import Timer`